extensions/amp-story-page-attachment/0.1/test/test-amp-story-open-page-attachment.js

```diff
@@ -205,6 +205,25 @@ describes.realWin(
       expect(openAttachmentLabelEl.textContent).to.equal('Custom text');
     });
 
+    it('should use the correct outlink text', () => {
+      const firstPage = win.document.createElement('amp-story-page');
+      storyEl.insertBefore(firstPage, page);
+      const otherOutlink = document.createElement('amp-story-page-outlink');
+      otherOutlink.appendChild(win.document.createElement('a'));
+      firstPage.appendChild(otherOutlink);
+
+      outlinkEl.querySelector('a').textContent = 'Custom text';
+      otherOutlink.querySelector('a').textContent = 'Wrong text';
+
+      outlink.buildCallback();
```

Comment:
should we `await` these so that there's not a race? (not sure if it matters here)

Reply:
It's not necessary here, none of the other tests do that, and buildCallback and layoutCallback are both synchronous so they don't have to be awaited.